build.gradle

```diff
@@ -25,20 +25,18 @@ import org.ajoberstar.gradle.git.tasks.*
 buildscript {
     repositories {
         mavenCentral()
-        maven {
-            url 'http://dl.bintray.com/jfrog/jfrog-jars'
-        }
+        jcenter()
```

Comment:
Dug a little deeper. There seems to be a fair amount of hostility between Maven Central and jCenter, probably because their attached business entities are competitors. jCenter is supposedly proxying Maven Central but the details of how well (new packages not being available for a while) are apparently up for debate.

I'll make our config consistent with jcenter placed first everywhere since it sounds like they might have better performance etc.